docs/import/add_disaster.html

```diff
@@ -15,10 +15,13 @@
   <script src="https://www.gstatic.com/firebasejs/7.2.1/firebase-auth.js"></script>
   <!-- Load jQuery, a 3rd-party library. -->
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.4/jquery-ui.js"></script>
+
   <script src="https://apis.google.com/js/api.js"></script>
   <script type="module" src="./add_disaster_startup.js"></script>
 
   <!-- Load our styles. -->
+  <link rel="stylesheet" href="./style.css">
```

Comment:
I think later stylesheets override earlier ones. Might be better to put the page-specific css last?

Reply:
Done.